submissions/DmitryHniezdilov/task-a-tiny-JS-world/index.js

```diff
@@ -0,0 +1,63 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/DmitryHniezdilov/task-a-tiny-JS-world/tree/gh-pages
+   Web app: https://dmitryhniezdilov.github.io/task-a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+
+const dog = {
+  species: "dog",
+  name: "Togo",
+  gender: "male",
+  legs: 4,
+  hands: 0,
+  saying: "woof-woof!",
+};
+
+const cat = {
+  species: "cat",
+  name: "Murka",
+  gender: "female",
+  legs: 4,
+  hands: 0,
+  saying: "meow-meow!",
+};
+
+const bear = {
+  species: "character",
+  name: "Winnie",
+  gender: "male",
+  legs: 4,
+  hands: 0,
+  saying: "Think, think, think!",
+};
+
+const man = {
+  species: "human",
+  name: "Artur",
+  gender: "male",
+  legs: 2,
+  hands: 2,
+  saying: "Do not borrow tomorrow's troubles today.",
+};
+
+const woman = {
+  species: "human",
+  name: "Marilyn",
+  gender: "female",
+  legs: 2,
+  hands: 2,
+  saying: "A career is born in public – talent in privacy.",
+};
+
+const listOfInhabitants = [dog, cat, bear, man, woman];
+
+// ======== OUTPUT ========
+
+listOfInhabitants.forEach((item) => {
+  const separator = "; ";
+  const listOfValues = Object.values(item).join(separator);
```

Comment:
Okay you're almost there. Task specifies not to use methods like `Object.values` and similar to them. Using them, you cannot rely on properties' order and presence. You can specify properties by hand, though.

Reply:
THX for the correction. I hope that it's ok.

Reply:
This code I updated only in the work branch in the current repo :
https://github.com/DmitryHniezdilov/task-a-tiny-JS-world/blob/task-a-tiny-JS-world/index.js.